providers/inwx/inwxProvider.go

```diff
@@ -59,11 +53,18 @@ var features = providers.DocumentationNotes{
 	providers.CanAutoDNSSEC:          providers.Unimplemented("Supported by INWX but not implemented yet."),
 	providers.DocOfficiallySupported: providers.Cannot(),
 	providers.DocDualHost:            providers.Can(),
-	providers.DocCreateDomains:       providers.Unimplemented("Supported by INWX but not implemented yet."),
+	providers.DocCreateDomains:       providers.Can("Does only create domain in nameserver and does not order domain."),
```

Comment:
Change to `providers.Can()`.  DocCreateDomains is not supposed to order the domain. It just adds it to the namespace.